pkg/store/storepb/rpc.proto

```diff
@@ -39,6 +40,21 @@ service Store {
   rpc LabelValues(LabelValuesRequest) returns (LabelValuesResponse);
 }
 
+/// WriteableStore reprents API against instance that stores XOR encoded values with label set metadata (e.g Prometheus metrics).
+service WriteableStore {
+  // WriteRequest allows you to write metrics to this store via remote write
+  rpc RemoteWrite(WriteRequest) returns (WriteResponse) {}
+}
+
+message WriteResponse {
+}
+
+message WriteRequest {
+  string tenant = 1;
+  int64 replica = 2;
```

Comment:
I'm curious how you think we should handle the zero value of this. I feel like we may need to either work with non-zero index or figure some other strategy out.

Reply:
Ack, with http, the zero value was inferred by checking if the header was present or not. Changing it to index-1 would be simple however, we should then ensure that the http path’s header also becomes 1-indexed to eliminate confusion.

Reply:
Is the replica header 0 indexed? I think 1 indexed would be the better option given the zero value situation.